### PR TITLE
README-Updates

### PR DESCRIPTION
**Pull request by vishalhcl-5960**
Added the info about SCA support & new fields of the ASE build steps.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 CHANGELOG
 =========
+1.3.0 (November, 2023)
+------
+- Support for new scan type : Software Composition Analysis (SCA) in AppScan on Cloud.
+- Support for adding Contact and description for AppScan Enterprise jobs.
```
Comment (kripajoym) [inline CHANGELOG.md:6]: D description

File: README.md
Comment (vinitasanghi) [inline README.md:1]: @vishalhcl-5960  please update the intro to mention SCA
Notes should be in the end I believe

**Review by kripajoym — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (vishalhcl-5960) [reply CHANGELOG.md]: Done.

File: README.md
Comment (vishalhcl-5960) [reply README.md:1]: Sure Vinita, I have shift both the notes to the end of the "Integration with HCL AppScan on Cloud/HCL AppScan 360°".

**Pull request merged**